### ISSUE TITLE
Some of the bugs that r yet to be fixed

### ISSUE THREAD

**Issue by tashi12** [Developer-Felix/Online-Job-Recruitment-System#1]
Hello,
Some of the bugs that r yet to be fixed are given below-
1) "About us"  is showing error.
2) Delete button on recruiter's dashboard is not working.

**Comment by Developer-Felix** [comment]
Can I fix these

**Comment by tashi12** [comment]
Yah..

**Comment by tashi12** [comment]
Also, upload resume has no existence in website..

**Comment by tashi12** [comment]
Hello sir,
r u fixed with the bugs thats occuring?

**Comment by Developer-Felix** [comment]
Yes, it is working.  Text me here please +254717713943 on WhatsApp for fast
response

**Comment by tashi12** [comment]
I don't use whattsup........
But i can mail you...

**Comment by Developer-Felix** [comment]
Okay have solved the resume error and added a new functionality of reset password

**Comment by Developer-Felix** [comment]
You can now pull the repo

**Comment by tashi12** [comment]
OkY..thanks....let me check

**Comment by tashi12** [comment]
Yah...its working now...
But showing "cv not available " on employers side..

**Comment by tashi12** [comment]
Also, resume value does not save in database
